supervisor/mock/src/main.rs

```diff
@@ -248,11 +241,13 @@ impl connector::Supervisor for MockSupervisor {
             &this.args.puppet_binary,
         )
         .arg("--transport")
-        .arg("auto_discover")
+        .arg("tcp")
+        .arg("--tcp-control-socket-addr")
+        .arg(SOCKET_ADDR_STR)
+        // .env("TML_CTRLSOCK_UNIXSEQPACKET", &control_socket_path)
```

Comment:
We'll need TCP control socket socket autodiscovery for e.g., QEMU targets anyways. For now passing in the socket address seems fine though.